index.js

```diff
@@ -106,7 +106,7 @@ function replaceSpecialCharacters(registryUri) {
 async function run() {
   // Get inputs
   const httpProxy = core.getInput(INPUTS.httpProxy, { required: false });
-  const maskPassword = core.getInput(INPUTS.maskPassword, { required: false }).toLowerCase() === 'true';
+  const maskPassword = (core.getInput(INPUTS.maskPassword, { required: false }).toLowerCase() || 'true') === 'true';
```

Comment:
Can we change this logic to only be `false` if maskPassword is explicitly set to `false`. Currently, if the customer writes something like `mask-password: asdf` by accident, the `mask-password` input becomes `false`.

Reply:
Updated.